### PR TITLE
Updated openshift-client task docs (serviceaccount, examples, etc)

### PR DESCRIPTION
**Pull request by siamaksade**
# Changes

* Simplified provided examples
* Added more docs on how to use the tasks

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: openshift-client/README.md [hunk 1]
```diff
@@ -1,118 +1,77 @@
...
-
-    ```
-    oc apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/openshift-client/taskUsingClusterResource/ocpipelinerun.yaml 
-    ```
-
-5. Check the logs of the pipelinerun by accessing the logs of the freshly created pods(Each task have one Pod).
\ No newline at end of file
+* **ARGS:** args to execute which are appended to `oc` e.g. `start-build myapp` (_default_: `help`)
+
+### Resources
```
Comment (vdemeester) [inline openshift-client/README.md]: The resource is only on one of the two task, most likely we should split the two task descriptions from each other :angel:

File: openshift-client/README.md [hunk 2]
```diff
@@ -1,118 +1,77 @@
...
+
+* **cluster**: a `cluster`-type `PipelineResource` specifying the target OpenShift cluster to execute the commands against it (only for `openshift-client-kubecfg` task) 
+
+## ServiceAccount
+
+If you don't specify a service account to be used for running the `TaskRun` or `PipelineRun`, the `default` [service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server). OpenShift by default does not allow the default service account to modify objects in the namespace. Therefore you should either explicitly grant permission to the default service account (by creating rolebindings) or create a new service account with sufficient privileges and specify it on the `TaskRun` or `PipelineRun`.
```
Comment (vdemeester) [inline openshift-client/README.md]: Shouldn't we document both (aka creating a specific service account or editing the default one), and recomend for the specific one ?

File: openshift-client/openshift-client-kubecfg-task.yaml [hunk 3]
```diff
@@ -2,19 +2,19 @@ apiVersion: tekton.dev/v1alpha1
...
-  namespace: default
 spec:
   inputs:
     resources:
-      - name: clusterdetails
+      - name: cluster
         type: cluster
     params:
-      - name: args
+      - name: ARGS
```
Comment (vdemeester) [inline openshift-client/openshift-client-kubecfg-task.yaml]: nit: I wonder if we need params to be all uppercase :angel:

Comment (siamaksade) [reply openshift-client/openshift-client-kubecfg-task.yaml]: There seems to be a convention for params to be all caps when I checked the rest of them in the catalog

File: openshift-client/README.md (hunk 1 above)
Comment (siamaksade) [reply openshift-client/README.md]: Updated

File: openshift-client/README.md (hunk 2 above)
Comment (siamaksade) [reply openshift-client/README.md]: I think it's enough to point to the right direction. Added links to Tekton and Kubernetes docs.

File: openshift-client/openshift-client-kubecfg-task.yaml (hunk 3 above)
Comment (vdemeester) [reply openshift-client/openshift-client-kubecfg-task.yaml]: fair enough

**Pull request merged**